packages/frontend/vite.config.ts

```diff
@@ -27,6 +27,15 @@ const externalPackages = [
 				: id;
 		},
 	},
+	// tinyld가 특수 UTF-8 문자를 사용하므로 Vite 빌드 과정에서 제외하고 CDN을 통해 로드함.
```

Comment:
ここのコメントアウト韓国語だけど日本語にしなくて大丈夫？
```suggestion
	// tinyldが特殊なUTF-8文字を使うので、Viteのビルド過程で除外してCDN経由でロードする.
```

Reply:
これは放置